src/stories/Library/campaign/Campaign.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export type CampaignProps = {
+  reverse?: boolean;
+  ellipsis?: boolean;
+  title: string;
+};
+
+const Campaign: React.FunctionComponent<CampaignProps> = ({
+  reverse,
+  ellipsis,
+  title,
+}) => {
+  return (
+    <section
+      className={`campaign
+      ${reverse && "campaign--reverse"}
+      ${ellipsis && "campaign--ellipsis"}
+       `}
```

Comment:
Consider controlling class names with `clsx` like we do elsewhere in this project.